packages/protocol/StreamingScreenShotCache.ts

```diff
@@ -42,6 +43,12 @@ export const StreamingScreenShotCache = createStreamingCache<
     await PaintsCache.readAsync();
 
     const paintPoint = findMostRecentPaint(time);
+    if (!paintPoint || !paintPoint.paintHash) {
+      // Don't try to paint (or repaint) if the current time is before the first cached paint
+      update(undefined, 1, "before-first-paint");
+      resolve();
+      return;
+    }
```

Comment:
We weren't handling this case very well. The new repaint 04 test highlighted that.